index.html

```diff
@@ -289,6 +289,20 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact-form">
+        <div class="description-text">
+          I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.
+        </div>
+        <div class="form">
+          <form>
```

Comment:
- [ ] Although the previous reviewer missed to point this out which is key in your portfolio website and also the building block in your project requirement(s)
> Your form should have `action` and `method` attributes that foster integration with [formspree service](https://formspree.io/html) to help you get notified when a user makes a submission to your form. 
Please, kindly fix that also.